labeler/Cargo.toml

```diff
@@ -11,3 +11,4 @@ once_cell = "1"
 regex = "1"
```

Comment:
Now we can remove `regex` dependency.

Reply:
It's still used in https://github.com/rust-lang/glacier/blob/master/labeler/src/github.rs#L114-L138

Reply:
Oh, seems so. My bad!